packages/formstr-app/src/containers/Dashboard/FormCards/DuplicateFormPopover.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from "react";
+import { CopyOutlined } from "@ant-design/icons";
+import { isMobile, makeTag } from "../../../utils/utility";
+import { constructDraftUrl } from "./Drafts";
+import { Tag } from "@formstr/sdk/dist/formstr/nip101";
+import { Tooltip } from "antd";
+
+type Props = {
+  tags: Tag[];
+};
+
+const DuplicateFormPopover: React.FC<Props> = ({ tags }) => {
+  const saveAndOpen = (duplicatedTags: Tag[], newFormId: string) => {
+    const duplicatedForm = {
+      formSpec: duplicatedTags,
+      tempId: newFormId,
+    };
+
+    const existingDrafts = localStorage.getItem("formstr:draftForms");
+    let updatedDrafts = existingDrafts ? JSON.parse(existingDrafts) : [];
+
+    updatedDrafts = [duplicatedForm, ...updatedDrafts];
+
+    localStorage.setItem("formstr:draftForms", JSON.stringify(updatedDrafts));
+
+    window.open(
+      constructDraftUrl(duplicatedForm, window.location.origin),
+      "_blank"
+    );
+  };
+
+  const handleFullDuplicate = () => {
+    const newFormId = makeTag(6);
+    const duplicatedTags = tags.map((tag) => {
+      if (tag[0] === "d") return ["d", newFormId];
+      if (tag[0] === "settings") {
+        try {
+          const settings = JSON.parse(tag[1]);
+          return [
+            "settings",
+            JSON.stringify({ ...settings, formId: newFormId }),
```

Comment:
ahh cause you needed to add formId back, makes sense, i Wonder if should avoid this double use